docs/Running-Hadoop-MapReduce-on-Tachyon.md

```diff
@@ -3,99 +3,157 @@ layout: global
-    specifying
-    `/pathToTachyon/clients/client/target/tachyon-client-{{site.TACHYON_RELEASED_VERSION}}-jar-with-dependencies.jar`
-    as the argument. This will place the jar in the Hadoop DistributedCache, and is desirable only
-    if you are updating the Tachyon jar a non-trivial number of times.
+For simplicity, we will assume a Psuedo-distributed Hadoop cluster, and you can start it by running:
 
--   For those interested in the second option, please revisit the Cloudera guide for more assistance.
-    One must simply package the Tachyon jar in the `lib` subdirectory of the job jar. This option is
-    the most undesirable since for every change in Tachyon, we must recreate the job jar, thereby
-    incurring a network cost for every job by increasing the size of the job jar.
+```bash
+$ cd $HADOOP_HOME
+$ ./bin/stop-all.sh
+$ ./bin/start-all.sh
+```
 
-In order to make the Tachyon executables available to the JobClient, one can also install the
-Tachyon jar in the `$HADOOP_HOME/lib` directory, or modify `HADOOP_CLASSPATH` by changing `hadoop-env.sh` to:
+Configure Tachyon to use the local HDFS cluster as its under storage system. You can do this by
+modifying `conf/tachyon-env.sh` to include:
 
-    $ export HADOOP_CLASSPATH=/pathToTachyon/clients/client/target/tachyon-client-{{site.TACHYON_RELEASED_VERSION}}-jar-with-dependencies.jar
+```bash
+export TACHYON_UNDERFS_ADDRESS=hdfs://localhost:9000
+```
 
-This will allow the code that creates the Job and submits it to reference Tachyon if necessary.
+Start Tachyon locally:
 
-# Example
+```bash
+$ ./bin/tachyon-stop.sh
+$ ./bin/tachyon-start.sh local
+```
 
-For simplicity, we will assume a psuedo-distributed Hadoop cluster.
+You can add a sample file to Tachyon to run wordcount on. From your Tachyon directory:
 
-    $ cd $HADOOP_HOME
-    $ ./bin/stop-all.sh
-    $ ./bin/start-all.sh
+```bash
+$ ./bin/tachyon tfs copyFromLocal LICENSE /wordcount/input.txt
+```
 
-Because we have a psuedo-distributed cluster, copying the Tachyon jar into `$HADOOP_HOME/lib` makes
-the Tachyon executables available to both the TaskTrackers and the JobClient. We can now verify that
-it is working by the following:
+This command will copy the `LICENSE` file into the Tachyon namespace with the path
+`/wordcount/input.txt`.
 
-    $ cd $HADOOP_HOME
-    $ ./bin/hadoop jar hadoop-examples-1.0.4.jar wordcount -libjars /pathToTachyon/clients/client/target/tachyon-client-{{site.TACHYON_RELEASED_VERSION}}-jar-with-dependencies.jar tachyon://localhost:19998/X tachyon://localhost:19998/X-wc
+Now we can run a MapReduce job for wordcount.
 
-Where X is some file on Tachyon and, the results of the wordcount job is in the X-wc directory.
+```bash
+$ bin/hadoop jar hadoop-examples-1.2.1.jar wordcount -libjars /<PATH_TO_TACHYON>/clients/client/target/tachyon-client-{{site.TACHYON_RELEASED_VERSION}}-jar-with-dependencies.jar tachyon://localhost:19998/wordcount/input.txt tachyon://localhost:19998/wordcount/output
```

Comment:
Should specify that if tachyon client jar is distributed to all nodes, `-libjars` is not needed.